src/readers/F3DReaderBoilerPlate.h.in

```diff
@@ -0,0 +1,90 @@
+/**
+ * This file is auto-generated by CMake, do not edit directly.
+ */
+
+#ifndef F3D@F3D_READER_NAME@_h
+#define F3D@F3D_READER_NAME@_h
+
+#include "F3DReaderFactory.h"
+
+#if @F3D_READER_HAS_GEOMETRY_READER@
+#include <@F3D_READER_VTK_READER@.h>
+#endif
+
+#if @F3D_READER_HAS_SCENE_READER@
+#include <@F3D_READER_VTK_IMPORTER@.h>
+#endif
+
+#include <vtkVersion.h>
+#include <vtksys/SystemTools.hxx>
```

Comment:
should these two be in the boiler plate ?

Reply:
yes because the custom files are added inside the class scope, where we cannot include any header file unfortunately.

Reply:
ok, I guess thats fine.